website/react-magma-docs/src/pages/api/table.mdx

```diff
@@ -116,32 +118,31 @@ import {
 } from 'react-magma-dom';
 
 export function Example() {
-  return (
-    <Table hasVerticalBorders>
-      <TableHead>
-        <TableRow>
-          <TableHeaderCell>Column</TableHeaderCell>
-          <TableHeaderCell>Column</TableHeaderCell>
-          <TableHeaderCell>Column</TableHeaderCell>
-          <TableHeaderCell>Column</TableHeaderCell>
-        </TableRow>
-      </TableHead>
-      <TableBody>
-        <TableRow>
-          <TableCell>Lorem ipsum dolor sit amet consectetur</TableCell>
-          <TableCell>Lorem ipsum dolor</TableCell>
-          <TableCell>Lorem ipsum dolor</TableCell>
-          <TableCell>Lorem ipsum</TableCell>
-        </TableRow>
-        <TableRow>
-          <TableCell>Lorem ipsum dolor sit amet</TableCell>
-          <TableCell>Lorem ipsum dolor</TableCell>
-          <TableCell>Lorem ipsum dolor</TableCell>
-          <TableCell>Lorem ipsum</TableCell>
-        </TableRow>
-      </TableBody>
-    </Table>
-  );
+  return;
+  <Table tableTitle="Vertical borders table" hasVerticalBorders>
```

Comment:
This doesn't look right

Reply:
Op, it crashes the whole Table page
![image](https://github.com/user-attachments/assets/5f97509e-e29e-4885-9105-9fcad67387af)

Reply:
Thanks Prettier